biostar/forum/templates/post_view.html

```diff
@@ -40,7 +41,15 @@
 
 {% block sidebar %}
 
-    <div id="similar-feed" post_uid="{{ post.uid }}"></div>
+    {#  Cache every hour #}
+    {% cache 3600 sidebar %}
```

Comment:
the cache needs to be post-specific, we can't use the same cache for all posts, the same content will show up, should be something like

    {% cache 3600 similar-posts post.id %}